Assets/Scripts/Battle/BattleSystem/Action/Attack.cs

```diff
@@ -2,20 +2,46 @@
 
 public class Attack : BattleAction
 {
+    /// <summary>
+    /// The Base Damage the attack does.
+    /// </summary>
     [SerializeField]
     private int damage;
+
+    /// <summary>
+    /// The Recoil Damage the attack does.
+    /// </summary>
     [SerializeField]
     private int recoil;
+
+    /// <summary>
+    /// The cost of MP per Attack.
+    /// </summary>
     [SerializeField]
     private int mpCost;
 
+    /// <summary>
+    /// Element type of attack to apply.
+    /// </summary>
+    [SerializeField]
+    private ElementTypes.ElementType ElementType = ElementTypes.ElementType.Physical;
+
+
+    /// <summary>
+    /// The amount to heal the battle combatant.
+    /// </summary>
+    [SerializeField]
+    private int HealAmount;
```

Comment:
Raw variables should start with a lowercase letter, while properties start with uppercase.